PHPCI/Helper/BaseCommandExecutor.php

```diff
@@ -81,7 +81,14 @@ public function executeCommand($args = array())
 
         if ($this->quiet) {
             $this->logger->log('Executing: ' . $command);
+        } else {
+            /**
+             * show the command to execute in purple
+             * @link http://tldp.org/HOWTO/Bash-Prompt-HOWTO/x329.html
+             */
+            $this->logger->log([null, null, "\033[0;35mExecuting: {$command}\033[0m", null]);
```

Comment:
We normally don't place the vars in the string, can you change it to:
$this->logger->log([null, null, "\033[0;35mExecuting: " . $command . "\033[0m", null]);